frame/staking/src/lib.rs

```diff
@@ -1216,6 +1248,41 @@ pub mod pallet {
 	#[pallet::storage]
 	pub(crate) type StorageVersion<T: Config> = StorageValue<_, Releases, ValueQuery>;
 
+
+	// The next storage items collectively comprise the voter bags: a composite data structure
+	// designed to allow efficient iteration of the top N voters by stake, mostly. See
+	// `mod voter_bags` for details.
+
+	/// How many voters are registered.
+	#[pallet::storage]
+	pub(crate) type VoterCount<T> = StorageValue<_, u32, ValueQuery>;
```

Comment:
This must always be equal to `CounterForNominators` plus `CounterForValidators`, right? you should sprinkle this around as debug_assert!, specially in tests as a postcondition.

Reply:
https://github.com/paritytech/substrate/pull/9081/commits/fb5431e1782c60a2c59d0f7fa3a5c752c73bed35

Reply:
Lets name this `CountFor*` or something, which will make it backwards compat with the counted map impl